packages/agent/src/certificate.ts

```diff
@@ -161,72 +142,69 @@ export class Certificate {
     if (!lookup) {
       throw new Error(`Could not find subnet key for subnet 0x${d.subnet_id.toString('hex')}`);
     }
-    return Buffer.from(lookup);
+    return lookup;
   }
 }
 
-const DER_PREFIX = Buffer.from(
+const DER_PREFIX = hexToUint8(
```

Comment:
I'm a little confused here, it used to be Buffer.from('blah','hex') so the result would be a Buffer. 

But now this is instead UInt8Array, and that works fine?

Reply:
I also changed the one place that referenced it